client/public/index.html

```diff
@@ -24,12 +22,14 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Alumni Portal | SKCT</title>
```

Comment:
Keep it there, dont remove it from index.html, since this will be shown when the website's loading initially from the server(CDN probably)

Reply:
okay got it